sdks/python/apache_beam/runners/common.py

```diff
@@ -225,7 +225,7 @@ def process(context):
         else:
           self.dofn_process = lambda context: fn.process(context, *args)
 
-        class CurriedFn(core.DoFn):
+        class CurriedFn(core.OldDoFn):
```

Comment:
This is a runner code of the OldDoFn so not really a new usecase. I don't know why the tests didn't catch this but there shouldn't be any more usage of them.